docs/cli-existing-project.md

```diff
@@ -42,13 +42,24 @@ In this guide, you'll propose the deployment on the command line and then approv
   * [Foundry](https://book.getfoundry.sh/getting-started/installation)
   * [Yarn](https://classic.yarnpkg.com/lang/en/docs/install/), [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm), or [pnpm](https://pnpm.io/installation)
   * [Node Version >=16.16.0](https://nodejs.org/en/download). (Run `node -v` to see your current version).
+  * [Rust Compiler](https://www.rust-lang.org/tools/install)
 
-## 2. Update Foundry
+## 2. Install Sphinx Foundry Fork
+For now, Sphinx requires a [fork of Foundry](https://github.com/sphinx-labs/foundry) that resolves several bugs that conflict with Sphinx. We're working on getting these fixed in the official Foundry repo.
 
+If you are using macOS, you will first need to install the Xcode Command Line Tools:
 ```
 foundryup
+xcode-select --install
```

Comment:
Seems like `foundryup` shouldn't be in this code block given that the description above it only mentions XCode